subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -157,7 +157,7 @@ include::sample[dir="userguide/tasks/configureUsingBlock/groovy",files="build.gr
 include::sample[dir="userguide/tasks/configureUsingBlock/kotlin",files="build.gradle.kts[tags=configure]"]
```

Comment:
❌ We should also [revert sample `configureUsingVar`](https://github.com/gradle/gradle/blob/master/subprojects/docs/src/samples/userguide/tasks/configureUsingVar/groovy/build.gradle). The text above doesn't make any sense anymore.

Or we should change this to tell how to configure using `TaskProvider#configure` which is what the sample now do it.